src/mlpack/methods/decision_tree/decision_tree_main.cpp

```diff
@@ -180,13 +180,22 @@ static void mlpackMain()
     {
       arma::Row<double> weights =
           std::move(CLI::GetParam<arma::Mat<double>>("weights"));
-      model->tree = DecisionTree<>(trainingSet, model->info, labels,
-          numClasses, weights, minLeafSize, minimumGainSplit);
+      if (CLI::HasParam("print_training_error"))
```

Comment:
Picky style comment do you mind to use ``{}`` around the ``if/else`` condition, to improve the readability.

Reply:
Fixed in 2b9d68a59ea6ab64e673a3eaa7fca6f4aebb42b7